src/clients/twitch.ts

```diff
@@ -50,8 +51,36 @@ function registerTwitch(): void
             {
                 // Good idea to check if it exists or not, dunno.
                 if(!bridge.targetDiscordChannel)
-                {
                     throw new Error('Cannot find Discord channel.');
+
+                // If it was a response the following will exist on the object -
+                /**
+                    'reply-parent-display-name': 'testingaccount__',
+                    'reply-parent-msg-body': 'test',
+                    'reply-parent-msg-id': 'f279b175-7100-4486-bb96-c188ea102bbd',
+                    'reply-parent-user-id': '821973125',
+  '                 'reply-parent-user-login': 'testingaccount__',
+                 */
+                // There has to be a parent display name if it is a thread, so good way to check if it is
+                // since tmijs doesn't give us a like userState.isThread() like Discord.JS would >:(.
```

Comment:
Yep, that's correct - no booleans here; we're relying on fuzzy conditionals - essentially the presence of the key would indicate it

Reply:
si